sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+# Release notes Jul'24
+
+Release notes for commit range
+[d2817d6d317db1](https://github.com/intel/llvm/commit/d2817d6d317db1143bb227168e85c409d5ab7c82)
+...
+[ebb3b4a21b3b0e](https://github.com/intel/llvm/commit/ebb3b4a21b3b0e977f44434781729df7de83e436)
+
+## TODO
+
+commit https://github.com/intel/llvm/commit/9876e19f4ff387b35b0c98c7d62e5f50e6de187d
+    [SYCL][XPTI] 'queue_id' metadata feature refactoring (#13070)
+    bugfix?
+
+commit https://github.com/intel/llvm/commit/29b4d855fa1a378e89182795e0d368304c40c3f6
```

Comment:
@mmoadeli, the patch didn't update any tests, so it is hard to understand if it had user-visible impact. Could you please help understand if we should add this to release notes and if so which message should be used?

Reply:
It fixed `cmath_test.cpp` and `math_test.cpp` on Windows for CUDA. But this isn't a configuration available in CI at the moment so they weren't marked XFAIL.

This does have a user impact, I think a message like below would be suitable for the release notes:

> Fix using C++ math functions when targeting Nvidia on Windows